core/server/data/migration/fixtures/008/01-fix-sqlite-format.js

```diff
@@ -0,0 +1,80 @@
+var config = require('../../../../config'),
```

Comment:
i just noticed that i might need to rename the name to `fix-sqlite-pg-format` and the message as well.